src/AltinnCore/Common/Services/Implementation/ExecutionAppSI.cs

```diff
@@ -230,51 +154,14 @@ public byte[] GetServiceResource(string org, string service, string resource)
             return fileContent;
         }
 
-        /// <summary>
-        /// Returns the service metadata for a service.
-        /// </summary>
-        /// <param name="org">The Organization code for the service owner.</param>
-        /// <param name="service">The service code for the current service.</param>
-        /// <returns>The service metadata for a service.</returns>
+        /// <inheritdoc/>
         public ServiceMetadata GetServiceMetaData(string org, string service)
```

Comment:
Can you rename this as well or may be you have created an issue to rename these parameters everywhere?

Reply:
removed